lib/commands/build.js

```diff
@@ -20,6 +20,7 @@ module.exports = Command.extend({
     { name: 'watch', type: Boolean, default: false, aliases: ['w'] },
     { name: 'watcher', type: String },
     { name: 'suppress-sizes', type: Boolean, default: false },
+    { name: 'build-file-path', type: 'Path', default: 'tests/dummy/' },
```

Comment:
Should probably the full path to the file (we want the path to the file, not the path to the directory that contains the file).

```suggestion
    { name: 'build-file-path', type: 'Path', default: 'tests/dummy/ember-cli-build.js' },
```